py_src/ipyelk/diagram/elk_text_sizer.py

```diff
@@ -0,0 +1,68 @@
+"""Simple widget to get text size from DOM
+"""
+import asyncio
+import logging
+
+# Copyright (c) 2020 Dane Freeman.
+# Distributed under the terms of the Modified BSD License.
+import traitlets as T
+from ipywidgets import DOMWidget
+
+from .._version import EXTENSION_NAME, EXTENSION_SPEC_VERSION
+
+logger = logging.getLogger()
```

Comment:
all widgets have `self.log`

Reply:
added a note about this in contributing, but haven't looked for stuff yet... something like [semgrep](https://github.com/returntocorp/semgrep) would be rad...